templates/components/what/networking/production.hbs

```diff
@@ -36,12 +36,12 @@
                     </p>
                 </div>
                 <div class="four columns">
-                    <img src="/static/images/buoyant.svg" />
+                    <img src="/static/images/buoyant.svg" alt="buoyant" />
                 </div>
             </div>
             <div class="testimonial row">
                 <div class="four columns">
-                    <img src="/static/images/1aim.svg" />
+                    <img src="/static/images/1aim.svg" alt="laim" />
```

Comment:
```suggestion
                     <img src="/static/images/1aim.svg" alt="1aim" />
```

Reply:
Opps nice catch I did not see the diff with my fonts thanks, fixed in  c44b973f57ce48884a757ee46cb42fc1c0d417bf